c3d-apply/workflow/Snakefile

```diff
@@ -220,3 +220,20 @@ rule threshold_prob:
         ),
     script:
         './scripts/prob_thres.py'
+
+rule create_fcsv:
+    # MIGHT NEED TO BE FIXED, BUT STRUCTURE IS THERE
+    # TRIED TO MATCH AFIDS NAMING SCHEME
+    input:
+        seg=rules.threhsold_prob.output.prob_combined,
+        fcsv_template=config['fcsv_template']
+    output:
+        fcsv=bids(
```

Comment:
I haven't tried it, but I think you need to wrap this whole thing (including the `contrast=config["space"]` in an `expand`).

Reply:
This is something that we should try today. I haven't actually tested the snakemake workflow since I've been working with a single segmentatoin file just to get the code working.